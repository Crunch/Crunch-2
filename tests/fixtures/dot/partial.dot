{{##def.snippet:
<div>{{=it.name}}</div>
#}}

{{#def.snippet}}<div>{{=it.age}}</div>