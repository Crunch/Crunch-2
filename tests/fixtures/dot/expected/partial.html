<div>Jake</div><div>Jake who?</div>